definitions/npm/bser_v1.x.x/flow_v0.22.x-/bser_v1.x.x.js

```diff
@@ -0,0 +1,58 @@
+declare type $npm$bser$BserTypes =
```

Comment:
Also, maybe have a look at my slides for writing libdefs: 
https://github.com/ryyppy/flow-libdef-talk/blob/master/dist/flow-libdef-talk.pdf